actions/upload-packages-to-aws/action.yaml

```
@@ -0,0 +1,14 @@
+name: Upload Packages to S3
+inputs:
+  rush-projects:
+    description: A JSON array of { packageName, projectFolder} objects to upload
+    required: true
+  s3-bucket:
+    description: S3 bucket name
+    required: true
+  s3-path:
+    description: S3 bucket path, if not provided root is used
+    required: false
```

It seems these are never used separately, maybe they should be combined into one input?
```suggestion
  s3-address:
    description: bucket-name/optional/path/prefix
    required: true
```